test-utils/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "test-utils",
+  "amdName": "preactTestUtils",
+  "version": "0.1.0",
+  "private": true,
+  "description": "Test-utils for Preact",
+  "main": "dist/testUtils.js",
+  "module": "dist/testUtils.mjs",
+  "umd:main": "dist/testUtils.umd.js",
+  "source": "src/index.js",
+  "license": "MIT",
+  "types": "src/index.d.ts",
+  "scripts": {
```

Comment:
The scripts are not needed here. The ones in the root `package.json` are all that'S needed :+1: